openedx/core/djangoapps/heartbeat/runchecks.py

```diff
@@ -0,0 +1,45 @@
+from importlib import import_module
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+
+from .defaults import HEARTBEAT_DEFAULT_CHECKS,\
+    HEARTBEAT_EXTENDED_DEFAULT_CHECKS,\
+    HEARTBEAT_CELERY_TIMEOUT
+
+
+def runchecks(request):
+    """
+    Iterates through a tuple of systems checks,
+    then returns a key name for the check and the value
```

Comment:
It returns a dictionary: `{subsystem: {'status': boolean, 'message': str}, ... }`